### PR TITLE
Add new social RSS icon

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- Start of updating social icons to use the official brand assets (other icons will follow in new PR)
- Added new social RSS icon
- No hover state

## QA
- Pull code
- Run `./run serve --watch`
- Open local http://0.0.0.0:8101/patterns/icons/#social or open demo https://vanilla-framework-canonical-web-and-design-pr-2639.run.demo.haus/examples/patterns/icons/icons-social/  
- You should see a new social icon and inside a demo example of docs, as we can't serve this push due to docs being on a previous release of our framework

## Details
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2560

## Visual
Screenshot 2019-11-12 at 15 59 47

### CODE REVIEW

**Comment by barrymcgee** [comment]
The code looks ok but I can't verify it as the docs site uses the library via NPM. Not sure how @deadlight or @anthonydillon work around this. 

It would be good to have this documented in README.md or CONTRIBUTING.md

File: scss/_patterns_icons.scss [hunk 1]
```diff
@@ -37,8 +37,10 @@ $color-social-icon-foreground: $color-mid-x-light !default;
   @include vf-p-icon-youtube;
   @include vf-p-icon-canonical;
   @include vf-p-icon-ubuntu;
+  @include vf-p-icon-rss;
   @include vf-p-icon-sizes;
   @include vf-p-icon-in-button;
+  @include vf-p-icon-rss;
```
Comment (lyubomir-popov) [inline scss/_patterns_icons.scss]: isn't this a duplicate @include?

**Comment by lyubomir-popov** [comment]
@barrymcgee the only way I've been able to do it is to copy>paste portions of the rendered documentation into an example, where the current css styles it.

File: scss/_patterns_icons.scss (hunk 1 above)
Comment (kwm14) [reply scss/_patterns_icons.scss]: @lyubomir-popov ah yes, will remove 👍

**Comment by kwm14** [comment]
> The only way I've been able to do it is to copy>paste portions of the rendered documentation into an example, where the current CSS styles it.

Thanks, @lyubomir-popov  added it to existing social icons example and I can now see the new icon 👍

Screenshot 2019-11-12 at 14 58 01

@barrymcgee

**Pull request merged**